docs(interlink): add peer asn and vlan id for L2 and self hosted (#6807)
scaleway/docs-content: pages/interlink/quickstart.mdx

3. Choose a [region](/interlink/concepts/#region) for your InterLink. This must correspond to the Scaleway VPC you want to attach later.

4. Select a [physical connection type](/interlink/concepts/#connection-type) for your InterLink, either [hosted](/interlink/reference-content/overview/#hosted-interlinks) or [self-hosted](/interlink/reference-content/overview/#self-hosted-interlink).

> 5. Select the following options, depending on your physical connection type: 

Reviewer: ```suggestion
5. Select one of the following options, depending on your physical connection type: 
```

   <Tabs>
    
        <TabsTab label="Hosted">

>         1. Select a [partner](/interlink/concepts/#partner). To establish an InterLink connection via a partner, generally you must be a client of that partner, e.g. with your infrastructure in their datacenter, or on your own premises using the partner as a network provider. [Get help with choosing a partner](/interlink/reference-content/overview/#hosted-interlinks).

Reviewer: ```suggestion
        1. Select a [partner](/interlink/concepts/#partner). To establish an InterLink connection via a partner, you must be a client of that partner, e.g., by hosting your infrastructure in their datacenter, or by using the partner as your on‑premises network provider. [Get help with choosing a partner](/interlink/reference-content/overview/#hosted-interlinks).
```

>         2. Select the [location](/interlink/concepts/#location) for your InterLink. This represents the physical location where the connection to Scaleway's network will be established.

Reviewer 2: ```suggestion
        2. Select the [location](/interlink/concepts/#location) for your InterLink. This is the physical location where the connection to the Scaleway network will be established.
```

>         3. Choose a [bandwidth](/interlink/concepts/#bandwidth) amount. This represents the maximum bandwidth that will be available over the InterLink.

Reviewer: ```suggestion
        3. Choose a [bandwidth](/interlink/concepts/#bandwidth) level. This represents the maximum bandwidth that will be available over the InterLink.
```

        4. If you selected a partner offering L2 connectivity, enter your peer ASN, and optionally, a VLAN ID.
        </TabsTab>
        <TabsTab label="Self-hosted">

>         1. Select a [dedicated connection](/interlink/concepts/#dedicated-physical-connection). The InterLink will be hosted over this connection. You must have already provisioned this connection via the Scaleway sales team. [Contact sales](https://www.scaleway.com/en/contact-sales/) if you would like to provision a dedicated connection: this cannot be done via the Scaleway console or API.

Reviewer: ```suggestion
        1. Select a [dedicated connection](/interlink/concepts/#dedicated-physical-connection). The InterLink will be hosted over this connection. You must have already provisioned this connection via the Scaleway Sales team. [Contact Sales](https://www.scaleway.com/en/contact-sales/) if you would like to provision a dedicated connection: this cannot be done via the Scaleway console or API.
```

>         2. Choose a [bandwidth](/interlink/concepts/#bandwidth) amount. This represents the maximum bandwidth that will be available over the InterLink.

Reviewer 2: ```suggestion
        2. Choose a [bandwidth](/interlink/concepts/#bandwidth) amount. This is the maximum bandwidth  available over the InterLink.
```

>         2. Choose a [bandwidth](/interlink/concepts/#bandwidth) amount. This represents the maximum bandwidth that will be available over the InterLink.

Reviewer: ```suggestion
        2. Choose a [bandwidth](/interlink/concepts/#bandwidth) level. This is the maximum bandwidth that will be available over the InterLink.
```

>         3. Enter your peer ASN, and optionally, a VLAN ID.

Reviewer 2: ```suggestion
        3. Enter your peer ASN and, optionally, a VLAN ID.
```
